### PR TITLE
fix(virtualRepeat) Fix tests that were broken by recent ngAnimate cha…

### PR DESCRIPTION
**Pull request by kseamon**
…nges.

### CODE REVIEW

**Comment by kseamon** [comment]
@ThomasBurleson

**Comment by kseamon** [comment]
@jelbourn

File: src/components/virtualRepeat/virtualRepeater.spec.js [hunk 1]
```diff
@@ -92,12 +92,13 @@ describe('<md-virtual-repeat>', function() {
...
   });
 
-  xit('should render only enough items to fill the viewport + 3 (vertical, no md-item-size)', function() {
+  it('should render only enough items to fill the viewport + 3 (vertical, no md-item-size)', function() {
     repeater.removeAttr('md-item-size');
     createRepeater();
     scope.items = createItems(NUM_ITEMS);
     scope.$apply();
     $$rAF.flush();
+    $$rAF.flush();
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtualRepeater.spec.js]: Add a comment explaining the double flush?

**Comment by jelbourn** [comment]
LGTM mod my one comment

**Comment by ThomasBurleson** [comment]
@kseamon, @jelbourn - In our Karma tests, when I do something tricky with the flushing I try to comment why the sequence is needed. Any thoughts why the 2nd `$$rAF.flush()` is needed` ?

**Comment by kseamon** [comment]
Honestly, I verified that my $$rAF was not firing, and added this on a hunch and it works. I'm sure it has something to do with the recent ngAnimate change. Any guesses?

**Comment by kseamon** [comment]
Rebased

**Pull request closed without merging**